src/lib/utilities/handle-error.ts

```diff
@@ -36,6 +36,7 @@ export const handleError = (
     });
     // Re-throw error to prevent other code from attempting to render
     errors.set(error);
+    (error as any).report = false;
```

Comment:
I'd recommend maybe we get a little tighter (e.g. `error as { report: boolean }`).

Reply:
👍